tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -82,8 +82,8 @@ void test_width_estimation() {
         {TYPED_LITERAL(CharT, "\u2e80\x58"), 3},
         {TYPED_LITERAL(CharT, "\u303e\x58"), 3},
         {TYPED_LITERAL(CharT, "\u303f\x58"), 2},
-        {TYPED_LITERAL(CharT, "\u3040\x58"), 3},
-        {TYPED_LITERAL(CharT, "\ua4cf\x58"), 3},

```

Comment:
![image](https://github.com/microsoft/STL/assets/60953653/30a645d3-2c25-4606-88cb-3b7e427cf42b)

Reply:
Should I add comments for these change?